requirements.txt

```diff
@@ -1,4 +1,4 @@
-markdown-poster@ git+https://github.com/Lightning-AI/LAI-markdown-poster-Component.git@760ddecb73f65af19a6968cbc81df651dac586fe
+markdown-poster@ git+https://github.com/Lightning-AI/LAI-markdown-poster-Component.git@8a52c0bc939e04461c76b2a10a3136a5f8ee6f78
```

Comment:
Should we just use the `main` branch? @aniketmaurya Thoughts?

Reply:
Yeah, good point. We should be using `main`

Reply:
Done! Updated the file.